### ISSUE TITLE
Major Error

### ISSUE THREAD

**Issue by N1T3SLAY3R** [webbukkit/dynmap#1280]
2013-05-12 18:34:02 [SEVERE] [dynmap] Exception during render job: world=world, map=null
2013-05-12 18:34:02 [SEVERE] java.lang.NullPointerException
2013-05-12 18:34:02 [SEVERE]    at org.dynmap.hdmap.TexturePack.<init>(TexturePack.java:1058)
2013-05-12 18:34:02 [SEVERE]    at org.dynmap.hdmap.TexturePack.resampleTexturePack(TexturePack.java:1320)
2013-05-12 18:34:02 [SEVERE]    at org.dynmap.hdmap.TexturePackHDShader$ShaderState.<init>(TexturePackHDShader.java:108)
2013-05-12 18:34:02 [SEVERE]    at org.dynmap.hdmap.TexturePackHDShader.getStateInstance(TexturePackHDShader.java:261)
2013-05-12 18:34:02 [SEVERE]    at org.dynmap.hdmap.HDMapManager.getShaderStateForTile(HDMapManager.java:126)
2013-05-12 18:34:02 [SEVERE]    at org.dynmap.hdmap.IsoHDPerspective.render(IsoHDPerspective.java:1440)
2013-05-12 18:34:02 [SEVERE]    at org.dynmap.hdmap.HDMapTile.render(HDMapTile.java:102)
2013-05-12 18:34:02 [SEVERE]    at org.dynmap.MapManager$FullWorldRenderState.processTile(MapManager.java:659)
2013-05-12 18:34:02 [SEVERE]    at org.dynmap.MapManager$FullWorldRenderState.run(MapManager.java:601)
2013-05-12 18:34:02 [SEVERE]    at org.dynmap.MapManager$DynmapScheduledThreadPoolExecutor$1.run(MapManager.java:186)
2013-05-12 18:34:02 [SEVERE]    at org.dynmap.MapManager$DynmapScheduledThreadPoolExecutor$2.run(MapManager.java:202)
2013-05-12 18:34:02 [SEVERE]    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
2013-05-12 18:34:02 [SEVERE]    at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
2013-05-12 18:34:02 [SEVERE]    at java.util.concurrent.FutureTask.run(FutureTask.java:166)
2013-05-12 18:34:02 [SEVERE]    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:178)
2013-05-12 18:34:02 [SEVERE]    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:292)
2013-05-12 18:34:02 [SEVERE]    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
2013-05-12 18:34:02 [SEVERE]    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
2013-05-12 18:34:02 [SEVERE]    at java.lang.Thread.run(Thread.java:722)

2013-05-12 18:34:04 [SEVERE] [dynmap] Exception during render job: world=world_nether, map=org.dynmap.hdmap.HDMap@74143ad
2013-05-12 18:34:04 [SEVERE] java.lang.NullPointerException
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.TexturePack.<init>(TexturePack.java:1058)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.TexturePack.resampleTexturePack(TexturePack.java:1320)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.TexturePackHDShader$ShaderState.<init>(TexturePackHDShader.java:108)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.TexturePackHDShader.getStateInstance(TexturePackHDShader.java:261)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.HDMapManager.getShaderStateForTile(HDMapManager.java:126)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.IsoHDPerspective.render(IsoHDPerspective.java:1440)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.HDMapTile.render(HDMapTile.java:102)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.MapManager$FullWorldRenderState.processTile(MapManager.java:667)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.MapManager$FullWorldRenderState.run(MapManager.java:601)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.MapManager$DynmapScheduledThreadPoolExecutor$2.run(MapManager.java:202)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.FutureTask.run(FutureTask.java:166)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:178)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:292)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
2013-05-12 18:34:04 [SEVERE]    at java.lang.Thread.run(Thread.java:722)
2013-05-12 18:34:04 [SEVERE] [dynmap] Exception during render job: world=world, map=org.dynmap.hdmap.HDMap@406c86cd
2013-05-12 18:34:04 [SEVERE] java.lang.NullPointerException
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.TexturePack.<init>(TexturePack.java:1058)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.TexturePack.resampleTexturePack(TexturePack.java:1320)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.TexturePackHDShader$ShaderState.<init>(TexturePackHDShader.java:108)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.TexturePackHDShader.getStateInstance(TexturePackHDShader.java:261)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.HDMapManager.getShaderStateForTile(HDMapManager.java:126)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.IsoHDPerspective.render(IsoHDPerspective.java:1440)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.HDMapTile.render(HDMapTile.java:102)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.MapManager$FullWorldRenderState.processTile(MapManager.java:667)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.MapManager$FullWorldRenderState.run(MapManager.java:601)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.MapManager$DynmapScheduledThreadPoolExecutor$2.run(MapManager.java:202)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.FutureTask.run(FutureTask.java:166)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:178)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:292)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
2013-05-12 18:34:04 [SEVERE]    at java.lang.Thread.run(Thread.java:722)
2013-05-12 18:34:04 [SEVERE] [dynmap] Exception during render job: world=Arctic, map=org.dynmap.hdmap.HDMap@2f650e17
2013-05-12 18:34:04 [SEVERE] java.lang.NullPointerException
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.TexturePack.<init>(TexturePack.java:1058)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.TexturePack.resampleTexturePack(TexturePack.java:1320)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.TexturePackHDShader$ShaderState.<init>(TexturePackHDShader.java:108)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.TexturePackHDShader.getStateInstance(TexturePackHDShader.java:261)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.HDMapManager.getShaderStateForTile(HDMapManager.java:126)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.IsoHDPerspective.render(IsoHDPerspective.java:1440)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.hdmap.HDMapTile.render(HDMapTile.java:102)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.MapManager$FullWorldRenderState.processTile(MapManager.java:667)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.MapManager$FullWorldRenderState.run(MapManager.java:601)
2013-05-12 18:34:04 [SEVERE]    at org.dynmap.MapManager$DynmapScheduledThreadPoolExecutor$2.run(MapManager.java:202)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.FutureTask.run(FutureTask.java:166)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:178)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:292)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
2013-05-12 18:34:04 [SEVERE]    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
2013-05-12 18:34:04 [SEVERE]    at java.lang.Thread.run(Thread.java:722)
2013-05-12 18:34:05 [SEVERE] [dynmap] Exception during render job: world=world, map=null
2013-05-12 18:34:05 [SEVERE] java.lang.NullPointerException
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.hdmap.TexturePack.<init>(TexturePack.java:1058)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.hdmap.TexturePack.resampleTexturePack(TexturePack.java:1320)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.hdmap.TexturePackHDShader$ShaderState.<init>(TexturePackHDShader.java:108)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.hdmap.TexturePackHDShader.getStateInstance(TexturePackHDShader.java:261)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.hdmap.HDMapManager.getShaderStateForTile(HDMapManager.java:126)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.hdmap.IsoHDPerspective.render(IsoHDPerspective.java:1440)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.hdmap.HDMapTile.render(HDMapTile.java:102)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.MapManager$FullWorldRenderState.processTile(MapManager.java:659)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.MapManager$FullWorldRenderState.run(MapManager.java:601)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.MapManager$DynmapScheduledThreadPoolExecutor$1.run(MapManager.java:186)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.MapManager$DynmapScheduledThreadPoolExecutor$2.run(MapManager.java:202)
2013-05-12 18:34:05 [SEVERE]    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
2013-05-12 18:34:05 [SEVERE]    at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
2013-05-12 18:34:05 [SEVERE]    at java.util.concurrent.FutureTask.run(FutureTask.java:166)
2013-05-12 18:34:05 [SEVERE]    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:178)
2013-05-12 18:34:05 [SEVERE]    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:292)
2013-05-12 18:34:05 [SEVERE]    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
2013-05-12 18:34:05 [SEVERE]    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
2013-05-12 18:34:05 [SEVERE]    at java.lang.Thread.run(Thread.java:722)
2013-05-12 18:34:05 [SEVERE] [dynmap] Exception during render job: world=world_the_end, map=org.dynmap.hdmap.HDMap@2c61f08d
2013-05-12 18:34:05 [SEVERE] java.lang.NullPointerException
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.hdmap.TexturePack.<init>(TexturePack.java:1058)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.hdmap.TexturePack.resampleTexturePack(TexturePack.java:1320)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.hdmap.TexturePackHDShader$ShaderState.<init>(TexturePackHDShader.java:108)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.hdmap.TexturePackHDShader.getStateInstance(TexturePackHDShader.java:261)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.hdmap.HDMapManager.getShaderStateForTile(HDMapManager.java:126)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.hdmap.IsoHDPerspective.render(IsoHDPerspective.java:1440)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.hdmap.HDMapTile.render(HDMapTile.java:102)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.MapManager$FullWorldRenderState.processTile(MapManager.java:667)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.MapManager$FullWorldRenderState.run(MapManager.java:601)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.MapManager$DynmapScheduledThreadPoolExecutor$1.run(MapManager.java:186)
2013-05-12 18:34:05 [SEVERE]    at org.dynmap.MapManager$DynmapScheduledThreadPoolExecutor$2.run(MapManager.java:202)
2013-05-12 18:34:05 [SEVERE]    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
2013-05-12 18:34:05 [SEVERE]    at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
2013-05-12 18:34:05 [SEVERE]    at java.util.concurrent.FutureTask.run(FutureTask.java:166)
2013-05-12 18:34:05 [SEVERE]    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:178)
2013-05-12 18:34:05 [SEVERE]    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:292)
2013-05-12 18:34:05 [SEVERE]    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
2013-05-12 18:34:05 [SEVERE]    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
2013-05-12 18:34:05 [SEVERE]    at java.lang.Thread.run(Thread.java:722)
2013-05-12 18:34:06 [INFO] [HerobrineAI] WorldGuard plugin detected!
2013-05-12 18:34:10 [INFO] [FactionsPlus] Checking for updates ... 
2013-05-12 18:34:11 [INFO] [FactionsPlus] Up to date!
2013-05-12 18:34:29 [INFO] [36;1mN1T3SLAY3R[37;1m logged in from [36;1m127.0.0.1:58304[37;1m in world [31;22m"world"[37;1m (Entity #[36;1m45872[37;1m)[0;39m
2013-05-12 18:34:29 [INFO] [Sentry] Registered sucessfully with Vault: 12 groups found. The GROUP: target will function
2013-05-12 18:34:38 [INFO] [FactionsPlus] updated the cached reference to Essentials' instance: `Essentials vDev2.11.130`
2013-05-12 18:34:38 [INFO] N1T3SLAY3R issued server command: /dynmap fullrender world
2013-05-12 18:34:42 [INFO] N1T3SLAY3R issued server command: /dynmap
2013-05-12 18:34:42 [INFO] Unknown command. Type "help" for help.
2013-05-12 18:34:47 [INFO] N1T3SLAY3R issued server command: /dynmap cancelrender world
2013-05-12 18:34:50 [INFO] N1T3SLAY3R issued server command: /dynmap fullrender world

Basically when i type /dynmap fullrender it says the queue is in progress, so i try cancelling it it doesnt work/ i can repeatingly type that commadn without any effects happening. A am using version v1.7 alpha 2. I would link you to the map address but u need to sign in/have rank etc. to view it. if u have it coded so that its all good heres the link: http://darkrealms.mc-unite.com:8123/

If you need my config and stuff just request and i shall reply
